### PR TITLE
Update to readme

### PR DESCRIPTION
**Pull request by sabisw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
 # Project Change Manager
 
 ## Overview
-Project Change Manager is a project change management tool that will manage your project's SAS assets and set them to a known state.  Never make the same change twice, and view an audit table to track changes.  It is inspired by [Liquibase](https://www.liquibase.org/).
-Includes many examples to help learn how to make the best use out of this tool.
+Project Change Manager is a project change management tool. The Project Change Manager manages manage your project's SAS assets and sets them to a known state.  Do not make the same change twice and view an audit table to track changes.  The Project Change Manager is inspired by [Liquibase](https://www.liquibase.org/). It includes several examples on how to make the best use of the Project Chanage Manager.
```
Comment (ccastillo232) [inline README.md]: Duplicate words here: "manages manage"

File: README.md [hunk 2]
```diff
@@ -1,12 +1,11 @@
...
+It is currently being developed on Linux SAS remotely. It can be enhanced to run on Windows or locally with only minor modifications.
 
-Project Change Manager is a set of a SAS programs and macros that will read as input an XML changelog which lists the changesets (SAS programs) to be executed. Each changeset will have its log file redirected to a common log directory, and its execution will be logged in a databasechangelog data set. A changeset can be run only once. By reviewing the databasechangelog you can quickly gather information to determine the state of your SAS assets, and by reviewing the logs you can review the outcome of each changeset.
+Project Change Manager is a set of a SAS programs and macros that is read as input into an XML changelog. The XML changelog, which lists the changesets (SAS programs) to be executed. Each changeset has its log file redirected to a common log directory, and its execution will be logged in a databasechangelog data set. A changeset can run only once. By reviewing the databasechangelog, you can quickly gather information to determine the state of your SAS assets. By reviewing the logs, you can review the outcome of each changeset.
```
Comment (ccastillo232) [inline README.md]: Incomplete sentence: "The XML changelog, which lists the changesets (SAS programs) to be executed."  Getting rid of the comma and "which" fixes it.

File: README.md [hunk 3]
```diff
@@ -89,45 +91,43 @@ Here is an example of using Project Change Manager using gradle.
...
 
 ### Creating a changeset
-A changeset is a SAS program.  It can be as simple or complex as you make it.  A good principle to follow is to make sure you have lots of informative _put_ statements in case you need to review the logs.
+A changeset is a SAS program. A changeset can be as simple or complex as you make it. A good principle to follow is to make sure you have lots of informative put statements in case you need to review the logs.
 
-When a changeset is executed Project Change Manager will pass it a global macrovariable _g\_pcm\_root_ which can then be used in the changeset.  This is the root location of Project Change Manager as defined by the **PCM\_ROOT** environment variable.
+When a changeset is executed Project Change Manager will pass it a global macrovariable g_pcm_root, which can then be used in the changeset. This is the root location of Project Change Manager as defined by the **PCM_ROOT** environment variable.
```
Comment (ccastillo232) [inline README.md]: The slashes before the underscores are to make sure the underscores aren't interpreted as an emphasis when github renders the README.md markdown.

File: README.md (hunk 1 above)
Comment (sabisw) [reply README.md]: Completed.

File: README.md (hunk 3 above)
Comment (sabisw) [reply README.md]: Completed.

**Comment by sabisw** [comment]
@ccastillo232 I just updated per your changes. You should be able to see the new commit.

**Comment by sabisw** [comment]
I went ahead and made the necessary updates as you requested. You should be able to see the newest commit. If not, let me know.

Samantha Gidlow
Technical Writer ▪ Platform Programming Documentation
www.sas.com<http://www.sas.com/>

**Pull request merged**